File: iOS - Using Objective-C/day1/day1Task3_CopyingText/day1Task3_CopyingText/ViewController.m
//
//  ViewController.m
//  day1Task3_CopyingText
//
//  Created by JETSMobileLabMini5 on 17/12/2024.
//

#import "ViewController.h"

@interface ViewController ()
@property (weak, nonatomic) IBOutlet UILabel *labelData;
@property (weak, nonatomic) IBOutlet UITextField *textField;

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view.
}
- (IBAction)enterData:(id)sender {
    _labelData.text = _textField.text  ;
}


@end

File: RxSwift/Day4/Task2_CombineDotAssign/Task2_CombineDotAssign/ViewController.swift
//
//  ViewController.swift
//  Task2_CombineDotAssign
//
//  Created by Usef on 05/02/2025.
//

import UIKit
import Combine

class ViewController: UIViewController {
    @IBOutlet weak var titleLB: UILabel!
    
    @IBOutlet weak var titleTextField: UITextField!
   
    
    
    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view.
    }
    
    @IBAction func assignBtn(_ sender: UIButton) {
        let publisher = Just(titleTextField.text)
            .map({ "say: \($0 ?? "")" })
            .assign(to: \.text, on: titleLB)
    }
    
}

